.github/workflows/test-all-versions.pr.yml

```diff
@@ -16,22 +16,22 @@ jobs:
     env:
       PR_LABELS: ${{ toJSON(github.event.pull_request.labels.*.name) }}
     outputs:
-      args: ${{ steps.lerna-args.outputs.args }}
+      args: ${{ steps.npm-workspace-args.outputs.args }}
     steps:
       - name: Checkout
         uses: actions/checkout@v4
         # Need lerna to list all packages
       - name: Install lerna
         run: npm install -g lerna@6.6.2
```

Comment:
do we still need lerna? :thinking: 
Looks like the script is not using it anymore :thinking:

Reply:
You are right. I missed that. Thanks. Fixed in commit 93ae7b0f.